types/react-big-calendar/index.d.ts

```diff
@@ -268,7 +268,7 @@ export interface CalendarProps<TEvent extends object = Event, TResource extends
     onDoubleClickEvent?: (event: TEvent, e: React.SyntheticEvent<HTMLElement>) => void;
     onSelectEvent?: (event: TEvent, e: React.SyntheticEvent<HTMLElement>) => void;
     onSelecting?: (range: { start: stringOrDate; end: stringOrDate }) => boolean | undefined | null;
-    onRangeChange?: (range: Date[] | { start: stringOrDate; end: stringOrDate }) => void;
+    onRangeChange?: (range: Date[] | { start: stringOrDate; end: stringOrDate }, view: View) => void;
```

Comment:
I can follow the implementation but this is also missing from the docs, no? Would be really nice if you could contribute this to the docs as well.

Reply:
Sorry but I can't find a way to edit the docs. It seems that the page http://intljusticemission.github.io/react-big-calendar/examples/index.html#prop-onRangeChange isn't available on react-big-calendar repo